src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java

```diff
@@ -201,12 +201,12 @@ public void fromXML(final XMLDocument xmlDoc) {
             voice = xmlDoc.getNodeValue(CON_VOICE_EXPR);
             String voiceXStr = xmlDoc.getNodeValue(CON_VOICEX_EXPR);
             if (voiceXStr != null && voiceXStr.length() > 0) {
-                voiceX = Integer.parseInt(voiceXStr);
+                voiceX = Long.parseLong(voiceXStr);
             }
             fax = xmlDoc.getNodeValue(CON_FAX_EXPR);
             String faxXStr = xmlDoc.getNodeValue(CON_FAXX_EXPR);
             if (faxXStr != null && faxXStr.length() > 0) {
-                faxX = Integer.parseInt(faxXStr);
+                faxX = Long.parseLong(faxXStr);
```

Comment:
Related with my previous comment - we won't have to `parseLong`.

Reply:
Review comment has been addressed.